### DISCUSSION TITLE
article/start-to-write

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#76] [Announcements]
# article/start-to-write

这是一个由NotionNext生成的站点

https://docs.tangly1024.com/article/start-to-write

**Comment by Bailey-24** [comment]
我十分建议您先阅读《Notion快捷键》这篇文章
这篇文章无了

**Reply by SkysCrystal** [reply]
我目前能打开

**Comment by kaseeen** [comment]
zeabur云部署链接挂了
